meinberlin/assets/scss/components/_tile.scss

```diff
@@ -7,16 +7,14 @@
 
 .tile__head {
     position: relative;
-    min-height: 6em;
 }
 
 .tile__image {
     display: block;
+    height: 11.8em;
```

Comment:
Can we define this as `min-height`? It should do the same by default, but is more robust.